spotify_dl/spotify_dl.py

```diff
@@ -40,14 +40,25 @@ def spotify_dl():
                         default=['bestaudio/best'])
     parser.add_argument('-m', '--skip_mp3', action='store_true',
                         help='Don\'t convert downloaded songs to mp3')
-    parser.add_argument("--url", action="store_true", help="Convert playlist link to uri first")
+    parser.add_argument('-l', '--url', action="store",
+                        help="Spotify Playlist link URL")
 
     args = parser.parse_args()
 
     if args.version:
         print("spotify_dl v{}".format(VERSION))
         exit(0)
 
+    if os.path.isfile(os.path.expanduser('~/.spotify_dl_settings')):
```

Comment:
@alvierahman90 can you have `.spotify_dl_settings` changed to ` .spotify_dlrc` .. sorta inline with how the other *nix files save the config files http://stackoverflow.com/q/11030552/92837

Also @avinassh let me know if you're ok with this, will merge and release